# This will disable the setup.py features that attempt to install missing
# requirements using pip
--- a/setup.py
+++ b/setup.py
@@ -95,7 +95,7 @@ setup(
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=requirements,
+    # install_requires=requirements,
 
     # List additional groups of dependencies here (e.g. development
     # dependencies). You can install these using the following syntax,
@@ -111,7 +111,7 @@ setup(
                 'sadisplay'
             ],
     },
-    setup_requires=['setuptools_scm<6.0'],  # setuptools_scm>=6.0 doesn't support Python 3.5
+    # setup_requires=['setuptools_scm<6.0'],  # setuptools_scm>=6.0 doesn't support Python 3.5
     use_scm_version=True,
 
     # If there are data files included in your packages that need to be
